source/Gesture Recognizer/Assets/Plugins/GestureRecognizer/GesturesDataset.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+using PDollar = PDollarGestureRecognizer;
+using QDollar = QDollarGestureRecognizer;
+
+namespace GestureRecognizer
+{
+    [CreateAssetMenu(fileName = "GestureDataSet", menuName = "Gesture Recognizer/New Gesture Dataset", order = 1)]
```

Comment:
Add a constants class to store these strings and int.

Reply:
Removed "order" and "fileName", added constant in the same class for "menuName"